uint8 ROBOT_BASE=0
uint8 ROBOT_OTHER_ARM=1
uint8 OBJECT=2
uint8 PREVIOUS_TARGET=3
uint8 NOT_MOVING = 4
uint8 refFrame
geometry_msgs/Pose ee_pose
float64[] joint_pose